### PR TITLE
0004 - Get All Delivery Employees

### PR DESCRIPTION
**Pull request by gfitzp2002**

### CODE REVIEW

File: src/main/java/org/kainos/ea/cli/EmployeeRequest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    private String f_name;
+    private String l_name;
+    private double salary;
+    private String bank_acc_num;
+    private String ni_num;
+
+    @JsonCreator
+    public EmployeeRequest(
+            @JsonProperty("f_name") String f_name,
```
Comment (glenn-hamilton) [inline src/main/java/org/kainos/ea/cli/EmployeeRequest.java]: The variables aren't in the standard Java naming convention

File: src/main/java/org/kainos/ea/db/EmployeeDao.java [hunk 2]
```diff
@@ -38,4 +36,35 @@ public List<Employee> getAllEmployees() throws SQLException {
...
         return employeeList;
     }
+
+    public List<EmployeeRequest> getAllDeliveryEmployees() throws SQLException {
+
+        Connection c = databaseConnector.getConnection();
+
+        Statement st = c.createStatement();
+
+        ResultSet rs = st.executeQuery("SELECT * FROM employee" +
```
Comment (glenn-hamilton) [inline src/main/java/org/kainos/ea/db/EmployeeDao.java]: Can you change this so it doesn't use `SELECT *`

File: src/main/java/org/kainos/ea/api/EmployeeService.java [hunk 3]
```diff
@@ -10,8 +12,33 @@ public class EmployeeService {
...
+
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+
+            throw new FailedToGetEmployeeException();
+        }
+
+    }
+
+    public List<Employee> getAllEmployee() throws FailedToGetEmployeeException {
```
Comment (shaunganley) [inline src/main/java/org/kainos/ea/api/EmployeeService.java:30]: do you need this method as well?

File: src/main/java/org/kainos/ea/db/EmployeeDao.java [hunk 4]
```diff
@@ -38,4 +36,35 @@ public List<Employee> getAllEmployees() throws SQLException {
...
+
+            deliveryEmployeeList.add(employee);
+        }
+
+        return deliveryEmployeeList;
+    }
+
+
+
+
```
Comment (shaunganley) [inline src/main/java/org/kainos/ea/db/EmployeeDao.java:69]: remove whitespace

**Review by shaunganley — APPROVED** [review]

**Pull request merged**